projects/admin/src/app/record/detail-view/document-detail-view/holdings/serial-holding-item/serial-holding-item.component.html

```
@@ -3,33 +3,33 @@
 SPDX-FileCopyrightText: UCLouvain
 SPDX-License-Identifier: AGPL-3.0-or-later
 -->
-@if (item && ((item().metadata.issue && item().metadata.issue.status !== itemIssueStatus.DELETED) || item().metadata.type === 'provisional')) {
+@if (item && ((editableItem().metadata.issue && editableItem().metadata.issue.status !== itemIssueStatus.DELETED) || editableItem().metadata.type === 'provisional')) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)